macros/utils/cross_db_utils/edr_day_of_week_mod_7.sql

```diff
@@ -0,0 +1,11 @@
+{% macro edr_day_of_week_mod_7(date_expr) %}
```

Comment:
I'm 90% sure that all the DBs have a syntax for extracting the day of week. 
Try this in Snowflake:
`SELECT DAYOFWEEK(cast('2022-08-02' as date))`
This will return a consistent number.